cmd/main.go

```diff
@@ -133,4 +145,15 @@ func main() {
 	err = <-errs
 	obs.Cancel(context.Background())
 	log.Fatal("Observation terminated: ", err)
+	return
+}
+
+func checkArgs(cmd *cobra.Command, args []string) bool {
```

Comment:
Do we need this?

Reply:
Yes, to check that an argument is provided for each command

Reply:
![image](https://github.com/absmach/coap-cli/assets/28790446/6acd6b44-e8c8-4703-ad4e-a5350b2a75fe)

No need for this